modules/installation-osp-creating-bootstrap-machine.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+[id="installation-osp-creating-bootstrap-machine_{context}"]
+= Creating the bootstrap machine
+
+The playbook sets the allowed address pairs on each port attached to our OpenShift nodes.
+
+Since the keepalived-managed IP addresses are not attached to any specific server, Neutron would block their traffic by default. By passing them to --allowed-address the traffic can flow freely through.
+
+An additional Floating IP is also attached to the bootstrap port. This is not necessary for the deployment (and we will delete the bootstrap resources afterwards). However, if the bootstrapping phase fails for any reason, the installer will try to SSH in and download the bootstrap log. That will only succeed if the node is reachable (which in general means a floating IP).
+
+.Prerequisites
+* The infrastructure ID from the installation program's metadata file is set as an environment variable (`$INFRA_ID`)
```

Comment:
In fact, `common.yaml` will read `$INFRA_ID` from `metadata.json`, which is in turn generated by the installer

Reply:
cf https://github.com/openshift/installer/blob/25fe53340d9a45f7979537989d7d850bdfacc301/upi/openstack/common.yaml#L5

Reply:
Excellent!